### PR TITLE
Fix order of tar args

### PR DESCRIPTION
**Pull request by NiLuJe**
Otherwise, current tar versions abort

### CODE REVIEW

**Comment by NiLuJe** [comment]
Sneaked in a fix for an (old) regression for a (weird) bug.

File: Makefile [hunk 1]
```diff
@@ -186,8 +186,8 @@ kindleupdate: all
 	# note that the targz file extension is intended to keep ISP from caching
 	# the file, see koreader#1644.
 	cd $(INSTALL_DIR) && \
-		tar czafh ../koreader-$(DIST)-$(MACHINE)-$(VERSION).targz \
-		-T koreader/ota/package.index --no-recursion
+		tar --no-recursion -czahf ../koreader-$(DIST)-$(MACHINE)-$(VERSION).targz \
```
Comment (Frenzie) [inline Makefile]: Might be clearer to write `-czahf --no-recursion` instead?

Comment (NiLuJe) [reply Makefile]: I have a tendency to like having the tarball filename directly following the 'f' arg, but maybe that's just me?

Comment (NiLuJe) [reply Makefile]: Noope, that's not just me, that's tar itself ^^.

Comment (NiLuJe) [reply Makefile]: That's with tar 1.30, FWIW.

Comment (Frenzie) [reply Makefile]: Sure, good point. But, and this is just my opinion, I think in that case it's clearer to write something like `-czah --no-recursion -f <filename>`.

Comment (NiLuJe) [reply Makefile]: Appears to work, and I have no objection/preference, so, let's :).

**Pull request merged**